tooling/minimize/src/rvalue.rs

```diff
@@ -190,6 +190,20 @@ pub fn translate_rvalue<'cx, 'tcx>(
             let ls = list![op; c];
             ValueExpr::Tuple(ls, ty)
         }
+        rs::Rvalue::Cast(rs::CastKind::Pointer(rs::adjustment::PointerCast::ReifyFnPointer), func, _) => {
+            let rs::Operand::Constant(box f1) = func else { panic!() };
+            let rs::ConstantKind::Val(_, f2) = f1.literal else { panic!() };
+            let rs::TyKind::FnDef(f, substs_ref) = f2.kind() else { panic!() };
+            let key = (*f, *substs_ref);
+
+            if !fcx.cx.fn_name_map.contains_key(&key) {
+                let fn_name = fcx.cx.fn_name_map.len();
+                let fn_name = FnName(Name::from_internal(fn_name as _));
+                fcx.cx.fn_name_map.insert(key, fn_name);
+            }
```

Comment:
Can you add a method to `cx` that encapsulates this, and use it everywhere that we currently directly work with `fn_name_map`?